react/Section/Section.js

```diff
@@ -22,8 +22,8 @@ export default function Section({
   header,
   pullout,
   slim,
-  tone,
-  level,
+  tone = '',
+  level = '',
```

Comment:
Is defaulting here helps achieve anything?

Reply:
Yeah I get flow errors in my app without defaulting to an empty string here

Reply:
If you're setting a default prop for `tone`, then you shouldn't have to declare it as an optional type. Might be why flow is throwing an error.

Reply:
This is not a proper fix because it will be converted to a required prop and will be throwing console warnings. We've found the issue in the flow config in profile ui repo

Reply:
I usually only make props optional if I haven't declared a default prop. Otherwise it's technically not really required since it has a fallback.

Reply:
I'm not seeing any console warnings...

Reply:
@craigcollie I agree - it's not really required. You don't need to pass anything to satisfy the contract with the component

Reply:
then it's an optional, right?